docs/operations/hotplug_interfaces.md

```diff
@@ -79,27 +79,86 @@ Please refer to the
+          bridge: {}
+    networks:
+    - name: defaultnetwork
+      pod: {}
+      # new network
+    - name: fancy-net
+      multus:
+        networkName: new-fancy-net
+ ...
 ```
-This will add the interface and network to the VM spec template and to the running VMI object.
 
-**NOTE**: You can use the `--help` parameter for more information on each
-parameter.
+The interface and network will be removed from the corresponding VMI object as well by Kubevirt controller.
+
+Alternatively, use JSON patch to automating the process: 
+```bash
+kubectl patch vm vm-fedora --type=json -p='[{"op": "add", "path": "/spec/template/spec/domain/devices/interfaces/-", "value": {"name": "new-fancy-net", "bridge": {}}}, {"op": "add", "path": "/spec/template/spec/networks/-", "value": {"name": "new-fancy-net", "multus": {"networkName": "new-fancy-net"}}}]'
+```
 
 You can now check the VMI status for the presence of this new interface:
 ```bash
 kubectl get vmi vm-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
 ## Removing an interface from a running VM
-Following the example above, the user can request an interface unplug operation. Please refer to the following snippet:
+Following the example above, the user can request an interface unplug operation
+by editing the VM spec template and set the desired interface state to 'absent':
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+template:
+  spec:
+    domain:
+      devices:
+        interfaces:
+          - name: defaultnetwork
+            masquerade: {}
+          # set the interface state to absent 
+          - name: new-fancy-net
+            state: absent
+            bridge: {}
+    networks:
+      - name: defaultnetwork
+        pod: {}
+      - name: fancy-net
+        multus:
+          networkName: new-fancy-net
+```
+The interface in the corresponding VMI object will be set with state 'absent' as well by Kubevirt controller.
+
+Alternatively, use JSON patch to automate the process:
 ```bash
-virtctl removeinterface vm-fedora --name dyniface1
+kubectl patch vm vm-fedora --type=json -p='[{"op": "replace", "path": "/spec/template/spec/domain/devices/interfaces/1/state", "value": "absent"}]'
+```
+Since JSON patch does not support pointing an element by attribute, the index of the subject
+interface should be specified explicitly, in this example its '1'.
+
+You can use 'jq' to get the interface index by its name:
+```bash
+iface_name="new-fancy-net"; 
+idx=$(kubectl get vm vm-fedora -o json | jq ".spec.template.spec.domain.devices.interfaces | map(.name) | index(\"${iface_name}\")")
+kubectl patch vm vm-fedora --type=json -p='[{"op": "replace", "path":  "/spec/template/spec/domain/devices/interfaces/'${idx}'/state", "value": "absent"}]'
 ```
-The subject interface state will be set as absent in the VM spec template and the running VMI object.  
 
 >**Note**: Existing VMs from version v0.59.0 and below does not support hot-unplug interfaces.
 
```

Comment:
Migration based hotplug/unplug-
This section has to be updated as well. It has `virtctl` examples.

Reply:
Done